README.md

```diff
@@ -1,14 +1,30 @@
 Notepad++ User Manual
 =======
 
-Notepad++ User Manual
-This document is dedicated to the countless lost hours of both people having to search for simple answers, and people having to answer questions that shouldn't have to be asked in the first place. Documentation is always hard to get, good documentation nearly impossible.
+***Documentation is like sex, when it's good, it's very, very good; when it's bad, it's better than nothing.***
+
+It's not our goal to make a bad document, but you've got the point.
```

Comment:
```diff
- It's not our goal to make a bad document, but you've got the point.
+ Of course, it's not our goal to create a bad documentation, but we are sure you've got the point to where we are heading.
```

note: as this is an internal place holder for internal doc team members, which will be changed in the near future, the content does not require much of our attention.

**only the typos within folder names are crucial now,**
to avoid later problems to rename the path names, as all depending files would have to be corrected as well,
(like with the typo at `notepad-plus-plus/PowerEditor/src/ScitillaComponent/`)